pom.xml

```diff
@@ -47,21 +73,32 @@
                         <configuration>
                             <createDependencyReducedPom>false</createDependencyReducedPom>
                             <minimizeJar>true</minimizeJar>
-
-                            <filters>
-                                <filter>
-                                    <artifact>github.scarsz:configuralize</artifact>
-                                    <excludes>
-                                        <!-- snakeyaml and json simple is already inside craftbukkit -->
-                                        <exclude>org/yaml/snakeyaml/**</exclude>
-                                        <exclude>org/json/simple/**</exclude>
```

Comment:
Was there a reason these excludes were removed?

Reply:
They are part of the filter for configuralize which isn't even being used to begin with

Reply:
Fair enough